benchmarks/viscoelastic_plastic_shear_bands/kaus_2010/kaus_2010_extension.prm

```diff
@@ -34,9 +37,23 @@ set Pressure normalization                 = no
 # Solver settings
 subsection Solver parameters
   subsection Stokes solver parameters
-    set Use direct solver for Stokes system = false
-    set Linear solver tolerance = 1e-7
-    set Number of cheap Stokes solver steps = 0
+    set Stokes solver type                  = block AMG
+    set Number of cheap Stokes solver steps = 2000
+    set Linear solver tolerance             = 1e-7
+    set GMRES solver restart length         = 50
+    set Use full A block as preconditioner  = true
+  end
+  subsection Newton solver parameters
+    set Max pre-Newton nonlinear iterations      = 5
+    set SPD safety factor                        = 0.9
+    set Nonlinear Newton solver switch tolerance = 1e-4
+    set Max Newton line search iterations        = 5
+    set Maximum linear Stokes solver tolerance   = 1e-7
+    set Use Newton residual scaling method       = false
+    set Use Newton failsafe                      = true
+    set Stabilization preconditioner             = SPD
+    set Stabilization velocity block             = SPD
+    set Use Eisenstat Walker method for Picard iterations = false
```

Comment:
Out of curiosity, what happens if you set this to `true`?

Reply:
Good question. The Eisenstat Walker method typically predicts a very coarse tolerance is required (ex: 1e0 or 1e-1), but even setting the Maximum linear solver tolerance to 1e-3 or 1-e4 can be problematic. Loosing the linear solver constraint and allowing it to evolve dynamically would certainly speed up the run, but for a benchmark I thought it would be good to keep the linear solver tolerance relatively strict. I am going to highlight this functionality in the updated continental extension cookbook.